### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by Helfull**

### CODE REVIEW

**Comment by Helfull** [comment]
@tweakimp @Matuiss2 please review this PR first before the dodging

File: jack_bot.py [hunk 1]
```diff
@@ -137,69 +140,6 @@ def __init__(self, debug=False):
         self.locations = []
         self.ordered_expansions = []
         self.building_positions = []
-        self.close_enemies_to_base = False
-        self.close_enemy_production = False
-        self.floating_buildings_bm = False
-        self.counter_attack_vs_flying = False
```
Comment (Matuiss2) [inline jack_bot.py]: this is missing from the new refactoring, its very important vs air untis

File: jack_bot.py [hunk 2]
```diff
@@ -231,28 +172,6 @@ def set_game_step(self):
...
             self.prepare_expansions()
             self.split_workers()
-
-        if self.ground_enemies:
-            for hatch in self.townhalls:
-                close_enemy = self.ground_enemies.closer_than(25, hatch.position)
-                close_enemy_flying = self.known_enemy_units.flying.closer_than(30, hatch.position)
-                enemies = close_enemy.exclude_type({DRONE, SCV, PROBE})
-                enemies_flying = close_enemy_flying.exclude_type(self.excluded)
-                if enemies_flying and not self.counter_attack_vs_flying:
```
Comment (Matuiss2) [inline jack_bot.py]: thats missing too

Comment (Matuiss2) [reply jack_bot.py]: everything related with self.counter_attack_vs_flying is missing on this pr

Comment (Helfull) [reply jack_bot.py]: should be back in, please check

**Pull request merged**